spec/requests/v2/create_claim_spec.rb

```diff
@@ -45,8 +45,14 @@ def run_background_jobs
 
```

Comment:
Re structured to keep atos examples separate (ready for removal in march time)

Added examples to test for correct routing of data